toolkit/tools/pkg/rpmssnapshot/rpmssnapshot.go

```diff
@@ -53,42 +52,38 @@ const (
 	rpmSpecBuiltRPMRegexDistributionIndex
 	rpmSpecBuiltRPMRegexArchitectureIndex
 	rpmSpecBuiltRPMRegexMatchesCount
+
+	chrootName = "rpmssnapshot_chroot"
+	runChecks  = false
 )
 
 type SnapshotGenerator struct {
-	chroot        *safechroot.Chroot
-	buildDirPath  string
-	workerTarPath string
+	simplechroottool.SimpleChrootTool
```

Comment:
Alternatively, if we want to make clear that it's an extension to `SimpleChrootTool`, we could do `type SnapshotGenerator simplechroottool.SimpleChrootTool`. But I think that's not the case - `simplechroottool.SimpleChrootTool` is a pre-defined chroot environment, while the generator is conceptually more than just an environment.

Reply:
One more thing: I think the generator has no reason to expose the `SimpleChrootTool` public functions. It's just supposed to be usable for the sole purpose of generating a snapshot. The user shouldn't be able to call arbitrary functions inside the chroot the generator is using. Similarly, they should have no need to access the chroot's root itself.

Reply:
While it's not a super common flow in our code base as far as I know this is a rasonablly common go idiom of adding an interface to a struct.

I'm not super tied to this, but my understanding is that this is fairly common go coding practice. The `type SnapshotGenerator simplechroottool.SimpleChrootTool` approach isn't very flexible since you can't add more context to the `SnapshotGenerator` type anymore, so I'm not a fan of that one. 

All that said, I don't mind moving them to an explicit member of the struct rather than an interface. I don't think it will be functionally different here.